src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -316,7 +317,7 @@ module.exports = function(
 
     if (oldState !== instance.state) {
       if (__DEV__) {
-        warning(
+        lowPriorityWarning(
```

Comment:
Can we leave these two (and one in ReactCompositeComponent) as hi-pri? They should be relatively rare anyway and are also more important than the others (IMO) since the breakage would be subtle.